content-elements/base/h1/prototype/styles.scss

```diff
@@ -1,20 +1,45 @@
 @import "../../../../styles/properties";
 
-@mixin styles($id: 'title-h1-piICx1', $title-font-family: $titleH1FontFamily, $title-font-size: $titleH1FontSize, $title-primary-color: $primary, $title-secondary-color: $secondary, $title-dark: $dark, $title-light: $light) {
+@mixin styles($id: 'title-h1-piICx1',
+  $title-font-family: $titleH1FontFamily,
+  $title-font-size: $titleH1FontSize,
+  $title-font-size-mobile: $titleH1FontSizeMobile,
```

Comment:
können wir da einen Fallback machen?
zB vor der Funktion:
$titleH1FontSizeMobile: $titleH1FontSize !default;

Reply:
können wir machen, aber wieso soll das bei diesem Wert passieren und bei allen anderen nicht? Was ist an diesem Wert anders?

Reply:
Mir ist gerade auch noch aufgefallen, dass in den Properties ein Wert hierfür gesetzt wurde und dort ein Fallback Wert gesetzt ist. Langt das dann nicht?